src/MySql/readme.md

```diff
@@ -115,6 +112,10 @@ directive:
       verb: New$|Update$
       variant: ^(?!.*?Expanded)
     hide: true
+  - where:
+      verb: Get$
+      subject: ^Configuration$|
```

Comment:
I think the tailing `|` was by mistake
```suggestion
      subject: ^Configuration$
```

Reply:
@isra-fel, Hi, I'm little running behind bc of workload but will make sure to finish these by EOW.. 
Thanks a lot for prompt reviews